canonical-web-and-design/maas-ui
# fix(ui): fix add chassis action names in http saga

## Done

- Renamed add-chassis action names to follow the `model/actionName` format generated by the machine slice

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to /MAAS/r/machines/chassis/add and fill out the form with incorrect values
- Submit the form and check that an http request is made, a spinner shows, and then error shows up at the top of the form

Fixes #2159
